strax/context.py

```diff
@@ -1652,6 +1654,7 @@ def add_method(cls, f):
 :param selection_str: Query string or sequence of strings to apply.
 :param keep_columns: Array field/dataframe column names to keep. 
     Useful to reduce amount of data in memory.
+:param drop_columns: Same but for dropping columns.
```

Comment:
_extemely minor_: perhaps be a bit more explicit (I usually don't read all the prams from top to bottom, only the ones that I suspect might be usefull